### DISCUSSION TITLE
LXT-010S-H

### DISCUSSION THREAD

**Discussion by mhet-albi** [Anime4000/RTL960x#323] [Q&A]
Hi all,

I am about to get into the rabbit hole of replacing ISP's gpon. So far I am gathering available information from different forums, issues and repos about available sticks. 

So far my candidates are _DFP-34X-2C3_ and _LXT-010S-H_. 
There is bunch of information about ODI, but not so much about LEOX device.  What is advantage of one over the other? As I see it, LEOX shoup have better fw and support, alas worse CPU and no option for 4port emulation. Which one would you prefer? 

Thanks!

**Comment by Anime4000** [comment]
Hi

Quick Rundown

| INFO       | DFP-34X-2C3 | LXT-010S-H |
|------------|-------------|------------|
| CPU        | RTL9601D    | RTL9601CI  |
| RAM        | 32MB        | 32MB |
| MEM        | 8MB         | 16MB |
| ME171 hack | Yes         | No |
| PPTP/VEIP  | Both        | PPTP only |
| SDK        | Later       | Latest    |
| 2.5G       | HiSGMII PHY/MAC, 2500Base-X (OC-48) | 2500Base-X (OC-48) |

If you want to use LXT-010S-H because of latest SDK/Linux, check your ONU how ME 171 is layout

if your ISP use ME171 and ME Point non `0x101`, you are out of luck, for this better use ODI stick

**Comment by mhet-albi** [comment]
This is output from current ONU. If I may ask, what does it state?

```##################################
MIB INFO:
         ME CLASS: 171
         DB NAME: extend_vlan, DBHandle: 68
##################################

<-----MeID[ 0xf201,61953 ], Addr[ 0x1b17a1]----->
         ass_type:02 
 oprtable_maxsize:00 00 
       input_tpid:81 00 
      output_tpid:81 00 
        down_mode:00 
         oprtable:[171_oprtable]
                  Mecls[171],Meid[0xf201,61953],Handle[67], Addr[0x001b171d]
                    f8 00 00 00 f8 00 00 00 00 0f 80 00 00 00 22 60 
                  Mecls[171],Meid[0xf201,61953],Handle[67], Addr[0x001b175d]
                    f8 00 00 00 e8 00 00 00 ff 0f ff ff ff 0f ff ff 
                  Mecls[171],Meid[0xf201,61953],Handle[67], Addr[0x001b177d]
                    e8 00 00 00 e8 00 00 00 ff 0f ff ff ff 0f ff ff 
                  Mecls[171],Meid[0xf201,61953],Handle[67], Addr[0x001b18e5]
                    f8 00 00 00 88 00 00 00 c0 0f 80 00 00 0f 80 00 
                  Table total num[4].

     p_associated:01 01 
 dscp2Pbitmapping:00 00 00 00 00 00 00 00 00 00 
                    00 00 00 00 00 00 00 00 00 00 
                    00 00 00 00 
---------------------------------------------------------------------
Total Record(s) num: 1```

**Reply by Anime4000** [reply]
based on parser here:
https://hack-gpon.org/gpon-omci-vlan-parser/

vlan 1100 ?

`p_associated:01 01` maybe 0x101 LAN 1?

**Comment by mhet-albi** [comment]
ME 84 also gives 1100. Does it mean, that LEOX stick should be ok? 
Is problem with ME171 related to FW or HW?
```
##################################
MIB INFO:
         ME CLASS: 84
         DB NAME: vlan_filter, DBHandle: 69
##################################

<-----MeID[ 0x0101,257 ], Addr[ 0x1b1909]----->
 vlan_filter_list:04 4c 
                          00 00 00 00 00 00 00 00 00 00 
                          00 00 00 00 00 00 00 00 00 00 
                          00 00 
      forward_opr:10 
           number:01 

<-----MeID[ 0x2081,8321 ], Addr[ 0x27b989]----->
 vlan_filter_list:00 00 
                          00 00 00 00 00 00 00 00 00 00 
                          00 00 00 00 00 00 00 00 00 00 
                          00 00 
      forward_opr:00 
           number:00 
---------------------------------------------------------------------
Total Record(s) num: 2
```
